src/TaskRunner/Commands/ComponentCheckCommands.php

```diff
@@ -173,7 +173,7 @@ function ($value) {
             $io->error([
                 'Failed the components check, please verify the report and update the project.',
                 'See the list of packages at',
-                'https://webgate.ec.europa.eu/fpfis/qa/package-reviews.',
+                'https://digit-dqa.prd.fpfis.tech.ec.europa.eu/package-reviews.',
```

Comment:
Final URL available please update: https://digit-dqa.fpfis.tech.ec.europa.eu/package-reviews.

Reply:
The URLs were updated